<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LabelEditor</class>
 <widget class="QMainWindow" name="LabelEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="imageView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
   </widget>
   <widget class="QGraphicsView" name="imageView_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="nextPB">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Next Frame</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitPB">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addPB">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Object</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>170</y>
      <width>311</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Object Adjustment Tools</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>140</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Object: </string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>135</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="removePB">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>80</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Object</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>717</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
